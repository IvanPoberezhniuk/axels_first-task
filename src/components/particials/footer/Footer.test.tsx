import { shallow, ShallowWrapper } from 'enzyme';

import { Footer } from '../../index';

let component: ShallowWrapper;
const setUp = () => shallow(<Footer />);

beforeEach(() => {
  component = setUp();
});

describe('Footer component', () => {
  it('SNAPSHOT', () => {
    expect(component).toMatchSnapshot();
  });
});

