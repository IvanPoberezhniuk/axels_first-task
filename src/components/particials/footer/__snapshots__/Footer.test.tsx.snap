// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component SNAPSHOT 1`] = `
<styled.footer>
  <small>
    Â© 2018-2019
  </small>
</styled.footer>
`;
